components/push/src/error.rs

```diff
@@ -71,6 +74,7 @@ impl ErrorKind {
             ErrorKind::StorageSqlError(_) => 29,
             ErrorKind::MissingRegistrationTokenError => 30,
             ErrorKind::TranscodingError(_) => 31,
+            ErrorKind::RecordNotFoundError(_, _) => 32,
```

Comment:
huh was there just no 32 before? Did we take it out leaving 33 alone or something? Weird.

Reply:
I'm pretty sure we removed it!